site/docs/components/navigation-item/index.mdx

```diff
@@ -6,11 +6,11 @@ data:
   description: "`NavigationItem` allows users to navigate between distinct applications and website pages."
   sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/src/navigation-item/NavigationItem.tsx"
   package:
-    name: "@salt-ds/lab"
+    name: "@salt-ds/core"
     initialVersion: "1.0.0-alpha.16"
   alsoKnownAs: ["Nav Item", "Link", "App button", "Header item"]
   relatedComponents: [{ name: "Pill", relationship: "similarTo" }]
   stickerSheet: ""
-  status: Lab component
+  status: Core component
```

Comment:
the other Core Components don't have this status, is this something new we're adding ?

Reply:
good point. I'll remove